flow/scripts/cts.tcl

```diff
@@ -24,21 +24,24 @@ proc save_progress {stage} {
   write_sdc $::env(RESULTS_DIR)/$stage.sdc
 }
 
+set cts_args [list -root_buf "$::env(CTS_BUF_CELL)" -buf_list "$::env(CTS_BUF_CELL)" \
+          -sink_clustering_enable \
+          -sink_clustering_size $cluster_size \
+          -sink_clustering_max_diameter $cluster_diameter \
+          -balance_levels]
+
 if {[info exist ::env(CTS_BUF_DISTANCE)]} {
-  clock_tree_synthesis -root_buf "$::env(CTS_BUF_CELL)" -buf_list "$::env(CTS_BUF_CELL)" \
-                      -sink_clustering_enable \
-                      -sink_clustering_size $cluster_size \
-                      -sink_clustering_max_diameter $cluster_diameter \
-                      -distance_between_buffers "$::env(CTS_BUF_DISTANCE)" \
-                      -balance_levels
-} else {
-  clock_tree_synthesis -root_buf "$::env(CTS_BUF_CELL)" -buf_list "$::env(CTS_BUF_CELL)" \
-                      -sink_clustering_enable \
-                      -sink_clustering_size $cluster_size \
-                      -sink_clustering_max_diameter $cluster_diameter \
-                      -balance_levels
+  lappend cts_args -distance_between_buffers "$::env(CTS_BUF_DISTANCE)"
+}
+
+if {[info exist ::env(CTS_ARGS)]} {
+  set cts_args $::env(CTS_ARGS)
 }
 
+puts "clock_tree_synthesis [join $cts_args " "]"
```

Comment:
Why print the command?  We don't elsewhere.

Reply:
This is the same as we do in global route.

Copy and paste actual args from log into config.mk and adjust.

Reply:
Where in global_route.tcl do you see this?

Reply:
Looks like you did that one as well: https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/commit/b8c7c7097fa6079c9fb3af397d3dca8ac412330a

I can live with it but I don't like the inconsistency in the flow.

Reply:
agreed, which is why I wrote it this way, consistent w detailed route where the arguments can be fully overridden.